python/private/python_repositories.bzl

```
@@ -191,6 +191,22 @@ def _python_repository_impl(rctx):
     elif rctx.attr.distutils_content:
         rctx.file(distutils_path, rctx.attr.distutils_content)
 
+    if "darwin" in platform:
```

In order to not attempt to use `install_name_tool` on platforms that don't have it (e.g. Debian), we ought to change this to do:
```suggestion
    if "darwin" in platform and "osx" == repo_utils.get_platforms_os_name(rctx):
```

This way this code path is going to only execute on OSX hosts when targeting the OSX toolchains.

For now this is the best we can do. This might also fix the Debian builds being broken.

---

Thanks, done in https://github.com/bazelbuild/rules_python/commit/078f566a44a6c10424332fee7b279e6fc86dbaa5.